datadog_checks_base/datadog_checks/base/checks/windows/perf_counters/base.py

```diff
@@ -155,6 +152,8 @@ def adopt_namespace(self, namespace):
             self.__NAMESPACE__ = old_namespace
 
     def cancel(self):
+        self.OBJECT_REFRESHER.remove_server(self._connection.server)
```

Comment:
What if multiple integration configurations have the same value of `server`?

Reply:
Then just decrements, no?